src/contentmanager.cpp

```diff
@@ -23,6 +23,7 @@ ContentManager::ContentManager(Library* library, kiwix::Downloader* downloader,
     setCurrentLanguage(QLocale().name().split("_").at(0));
     connect(mp_library, &Library::booksChanged, this, [=]() {emit(this->booksChanged());});
     connect(this, &ContentManager::remoteParamsChanged, this, &ContentManager::updateRemoteLibrary);
+    connect(this, &ContentManager::remoteParamsChanged, this, &ContentManager::updateLocalLibrary);
```

Comment:
It is a bit strange to connect `updateLocalLibrary` slot to `remoteParamsChanged` signal.
Maybe we could rename `remoteParamsChanged` to `filterParamsChanged` to better describe what the signal is about ?